pyGHDL/dom/NonStandard.py

```diff
@@ -80,8 +80,9 @@ def __init__(self):
         self.__ghdl_init()
 
     def __ghdl_init(self):
-        """Initialization: set options and then load libraries."""
-
+        """
```

Comment:
I think a online was here ok unless the docstring becomes longer.

Reply:
Changed.